BLUESPAWN-win-client/src/hunt/hunts/HuntT1136.cpp

```diff
@@ -41,6 +42,19 @@ namespace Hunts {
         }
         SUBSECTION_END();
 
+        SUBSECTION_INIT(HIDDEN_USER, Normal);
```

Comment:
Propose moving this to Cursory due to speed of check and high likelihood of maliciousness

Reply:
Ok